packages/ogmios/src/CardanoNode/OgmiosObservableCardanoNode/OgmiosObservableCardanoNode.ts

```diff
@@ -52,6 +62,15 @@ export class OgmiosObservableCardanoNode implements ObservableCardanoNode {
       distinctUntilChanged(isEqual),
       shareReplay({ bufferSize: 1, refCount: true })
     );
+    this.healthCheck$ = this.#connectionConfig$.pipe(
```

Comment:
Can't we use a potentially already existing connection?
```suggestion
    this.healthCheck$ = this.#interactionContext$.pipe(
```

Reply:
I found that it was conflicting in my test, since the WS connection would disconnect and then cause problems with the whole context, so thought it would be best to keep the two concerns separate. The health check just makes a HTTP GET request after all.